lib/quantity/math.ex

```diff
@@ -99,6 +110,7 @@ defmodule Quantity.Math do
     end)
     |> case do
       :error -> :error
+      {:error, :empty_list} -> {:error, :empty_list}
```

Comment:
fine 👍

Reply:
resolved in: https://github.com/wise-home/quantity/pull/84/commits/436b255fd736a9dfbed64a2790ccd72717929e55

> Two questions here:
> 
> 1. Is it possible to receive empty list error here when it should be already patter matched by the first clause?
> 2. When `add/2` encounters any error the reason will not be returned right? Maybe it is worth to also return the error reasons for the functions without exclamation mark?

I've changed it so that there are no longer returned any {:error, reason} but only :error as before.
So `add/2' and its siblings will only return {:ok, result} or error. while the bangers? ('!' functions) will raise.